libc-test/build.rs

```diff
@@ -2215,6 +2217,12 @@ fn test_freebsd(target: &str) {
             // should've been used anywhere anyway.
             "TDF_UNUSED23" => true,
 
+            // Removed in FreeBSD 15
+            "TDF_CANSWAP" | "TDF_SWAPINREQ" => true,
+
+            // Unaccessible in FreeBSD 15
+            "TDI_SWAPPED" | "P_SWAPPINGOUT" | "P_SWAPPINGIN" => true,
+
```

Comment:
Do you mind moving the freebsd-specific changes to a separate commit so cherry picking is a bit easier?

Reply:
you mean another PR ? usually we merge squashed commits ultimately.

Reply:
No, just separate commits. Usually only messy history needs to get squashed, and this specifically won't since it will get merged with other CI fixes.

(I'm just asking because I don't know what exactly needs to get cherry picked to the 0.2 branch to get that CI working)

Reply:
should be ok in theory,  l ll get to it soon-ish